content/fast/incorporate-performance-budgets-into-your-build-tools/codelab-setting-performance-budgets-with-webpack.md

```diff
@@ -0,0 +1,193 @@
+---
+page_type: glitch
+title: Setting performance budgets with webpack
+author: mihajlija
+description: |
+  Learn how to set performance budget and keep your bundlesize in check with 
+  webpack.
+web_updated_on: 2019-01-31
+web_published_on: 2019-01-31
+glitch: webpack-performance-budgets
+---
+
+[Webpack](https://developers.google.com/web/fundamentals/performance/webpack/)
+combines all your imported files and packages them into one or more output
+files known as bundles. Bundling is neat, but as your app grows your bundles
+will grow too. You need to monitor bundle sizes to ensure that they don’t grow
+too large and affect how long your application takes to load. Webpack supports
+setting [performance budgets](https://web.dev/fast/performance-budgets-101)
+based on **asset size** and it can keep an eye on bundle sizes for you.
+
+To see it in action, here’s an app that counts the days left until New Year’s.
+It’s built with [React](https://reactjs.org/) and [moment.js](https://momentjs.com/).
+(Just like real-world apps that increasingly rely on frameworks and libraries. 😉)
+
+![An app that counts the days left until New Year’s day](countdown-app.png)
+
+## Measure
+
+This Glitch already contains the app bundled with webpack.
```

Comment:
I wonder if we should say "this code lab" instead. Just in case folks aren't quite familiar with what we mean by "Glitch"